contracts/FlowStakingCollection.cdc

```diff
@@ -1005,100 +1060,123 @@ pub contract FlowStakingCollection {
+        let stakingCollectionRef = account.capabilities.borrow<&StakingCollection>(self.StakingCollectionPublicPath)
             ?? panic("Could not borrow ref to StakingCollection")
 
         return stakingCollectionRef.getDelegatorIDs()
     }
 
     /// Function to get all Node Info records for all Staking records in a users StakingCollection, if one exists.
-    pub fun getAllNodeInfo(address: Address): [FlowIDTableStaking.NodeInfo] {
+    access(all) fun getAllNodeInfo(address: Address): [FlowIDTableStaking.NodeInfo] {
         let account = getAccount(address)
 
-        let stakingCollectionRef = account.getCapability<&StakingCollection{StakingCollectionPublic}>(self.StakingCollectionPublicPath).borrow()
+        let stakingCollectionRef = account.capabilities.borrow<&StakingCollection>(self.StakingCollectionPublicPath)
             ?? panic("Could not borrow ref to StakingCollection")
 
         return stakingCollectionRef.getAllNodeInfo()
     }
 
     /// Function to get all Delegator Info records for all Delegation records in a users StakingCollection, if one exists.
-    pub fun getAllDelegatorInfo(address: Address): [FlowIDTableStaking.DelegatorInfo] {
+    access(all) fun getAllDelegatorInfo(address: Address): [FlowIDTableStaking.DelegatorInfo] {
         let account = getAccount(address)
 
-        let stakingCollectionRef = account.getCapability<&StakingCollection{StakingCollectionPublic}>(self.StakingCollectionPublicPath).borrow()
+        let stakingCollectionRef = account.capabilities.borrow<&StakingCollection>(self.StakingCollectionPublicPath)
             ?? panic("Could not borrow ref to StakingCollection")
 
         return stakingCollectionRef.getAllDelegatorInfo()
     }
 
     /// Global function to get all the machine account info for all the nodes managed by an address' staking collection
-    pub fun getMachineAccounts(address: Address): {String: MachineAccountInfo} {
+    access(all) fun getMachineAccounts(address: Address): {String: MachineAccountInfo} {
         let account = getAccount(address)
 
-        let stakingCollectionRef = account.getCapability<&StakingCollection{StakingCollectionPublic}>(self.StakingCollectionPublicPath).borrow()
+        let stakingCollectionRef = account.capabilities.borrow<&StakingCollection>(self.StakingCollectionPublicPath)
             ?? panic("Could not borrow ref to StakingCollection")
 
         return stakingCollectionRef.getMachineAccounts()
     }
 
     /// Determines if an account is set up with a Staking Collection
-    pub fun doesAccountHaveStakingCollection(address: Address): Bool {
+    access(all) fun doesAccountHaveStakingCollection(address: Address): Bool {
         let account = getAccount(address)
-
-        return account.getCapability<&StakingCollection{StakingCollectionPublic}>(self.StakingCollectionPublicPath).check()
+        return account.capabilities
+            .get<&StakingCollection>(self.StakingCollectionPublicPath)
+            .check()
```

Comment:
It should be 
`
...
?.check() ?? false
`
here imo - originally it was correct

Reply:
A recent FLIP is changing `account.capabilities.get` to not return an optional any more
https://github.com/onflow/flips/pull/242
So this is a change to be compatible with that update, which will be released in the next CLI version that should be coming out this week.

Reply:
hmm all right